example/main.py

```diff
@@ -186,5 +192,16 @@ def _get_result_from_dialogflow(text_to_be_analyzed: str) -> str:
         raise
     return '\n'.join(fulfillment_message.text.text[0] for fulfillment_message in response.query_result.fulfillment_messages)
 
+def upload_message_to_bigquery(json : dict):
+    # PROJECT_ID =os.getenv("BIGQUERY_PROJECT")
+    project_id = 'pycontw-225217'
```

Comment:
if you get a chance, would you use `os.getenv` in place of hard code `project_id`?
my bad for hard code the id during pycon sprint 😅

Reply:
Would `os.getenv`  be easier to maintain?

Reply:
yes, better than hardcode from my perspective (since we can change all of the bq-project-id without changing the code base)